src/openforms/config/models.py

```diff
@@ -134,6 +134,11 @@ class GlobalConfiguration(SingletonModel):
             "URL to the main website. Used for the 'back to municipality website' link."
         ),
     )
+    sdk_website = models.URLField(
+        _("sdk website link"),
+        blank=True,
+        help_text=_("URL to the SDK."),
+    )
```

Comment:
Please rename this specifically to "cancel_appointment_page". I'm also expecting that a global option will not be sufficient and there will be pages specific per form, so this may get a form-specific override in the future. Let's hold off on that for now though. Either way, forms will be embedded in many different CMS pages and there is no "single sdk website".

Help text should also be something like "URL to the page where the appointment cancellation should  happen."

The URL to the SDK is confusing since there's an SDK URL setting which points to which javascript bundle to load.